WRN-12780: Fix Scroller and VirtualList to focus the topmost element after scroll via voice control in pointer mode (#1118)
enactjs/sandstone: CHANGELOG.md

The following is a curated list of changes in the Enact sandstone module, newest changes on the top.

## [unreleased]

### Fixed

> - `sandstone/Scroller` and `sandstone/VirtualList` to not restore focus after scroll by voice control in pointer mode

Reviewer: ```suggestion
- `sandstone/Scroller` and `sandstone/VirtualList` to focus the topmost element after scroll by voice control in pointer mode
```

> - `sandstone/Scroller` and `sandstone/VirtualList` to not restore focus after scroll with voice control in pointer mode

Reviewer 2: ```suggestion
- `sandstone/Scroller` and `sandstone/VirtualList` to not restore focus after scroll by voice control in pointer mode
```
